### PR TITLE
Enumerable

### PR DESCRIPTION
**Pull request by Cabraham1**
In this milestone @kkolade @sntgopl  and I did the following implementation

Create a class MyList that has an instance variable @list.
In MyList implement a method #each that yields successive members of @list and uses the MyEnumerable module.
Create a module MyEnumerable that implements the following methods (they should have the same functionality as methods in [Enumerable](https://ruby-doc.org/core-3.0.0/Enumerable.html):
#all?
#any?
#filter
Each class and module should have a separate .rb file.
Verify your solution:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,247 +13,175 @@ After you're finished please remove all the comments and instructions!
...
-  - [Answer_2]
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<p align="right"><a href="#readme-top">👆</a></p>
 
 <!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
```
Comment (1GORDON) [inline README.md]: Great job team🤝

- [ ] Kindly fix the broken MIT license link to make your readme file more professional.

File: my_list.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+require_relative 'module'
+class Mylist
```
Comment (1GORDON) [inline my_list.rb]: Great job team🤝

- [ ] As stated in the project requirements, kindly ensure the `Mylist` class implements the `#each` method that yields all members of @list in succession.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @Cabraham1, @sntgopl, and @kkolade ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: module.rb [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    puts !oyin.empty?
+  end
+
+  def filter?
+    oyin = []
+    list.each do |said|
+      oyin.push(said) if yield(said)
+    end
+    puts oyin
+  end
```
Comment (OyePriscilla) [inline module.rb:28]: - [optional] Kindly remove the `?` added to the filter method, it implies that the method should either return `true` or `false`. I got this 👇 as a result of that. ℹ️ 

Untitled

**Review by OyePriscilla — APPROVED** [review]
Hi @Cabraham1, @sntgopl and @kkolade 👏 👏, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
Create a class and module in Ruby ✔️
`Mylist` class implements the `#each` method ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**